tools/clang/test/CodeGenSPIRV/intrinsics.length.hlsl

```diff
@@ -0,0 +1,28 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// According to HLSL reference:
+// The 'length' function can only operate on vector of floats.
+
+// CHECK:      [[glsl:%\d+]] = OpExtInstImport "GLSL.std.450"
+
+void main() {
+  float result;
+
+// CHECK:      [[a:%\d+]] = OpLoad %float %a
+// CHECK-NEXT: [[length_a:%\d+]] = OpExtInst %float [[glsl]] Length [[a]]
+// CHECK-NEXT: OpStore %result [[length_a]]
+  float a;
+  result = length(a);
```

Comment:
yeah. Since a scalar is a vec1, I thought we can call the function with the scalar. (e.g. length of a scalar x = x). But you're right, the spec doesn't specify the behavior for scalars. I'll remove the test case.